map_test.go

```diff
@@ -85,9 +85,9 @@ func testMap(t *testing.T, when spec.G, it spec.S) {
 
 		it("should return an ordering of buildpacks", func() {
 			m := lifecycle.BuildpackMap{
-				"buildpack1@version1.1": {Name: "buildpack1-1.1"},
-				"buildpack1@version1.2": {Name: "buildpack1-1.2"},
-				"buildpack2@latest":     {Name: "buildpack2"},
+				"buildpack1@version1.1": {Name: "buildpack1-1.1", Version: "1.1"},
```

Comment:
What does this change accomplish?

Reply:
Ensure the version is being passed to the output.

Reply:
The returned objects are just copies of the original objects (see `lookup`), so the name seemed sufficient. No strong opinion, but we might want to make the same changes in the `#ReadGroup` test.

Reply:
I guess I see your point. This change was simply one we chose to keep as we were writing tests to triage the bug. I'm OK either way.

Reply:
Would prefer to remove them then, to keep the test clear and consistent.